drivers/adc/adc_stm32.c

```diff
@@ -1431,14 +1281,12 @@ static const struct adc_stm32_cfg adc_stm32_cfg_##index = {		\
 	.has_temp_channel = DT_INST_PROP(index, has_temp_channel),	\
 	.has_vref_channel = DT_INST_PROP(index, has_vref_channel),	\
 	.has_vbat_channel = DT_INST_PROP(index, has_vbat_channel),	\
```

Comment:
I don't think we need these prop anymore. Do we ?

Reply:
I suppose we could remove them and make temp_channel & others default to -1 to prevent unwanted behavior

Reply:
Done